client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -147,167 +168,225 @@ export class DisplayTable extends React.Component {
-                                col_configs_with_defaults[col_key].formatter
-                              }
-                              content={total_row[col_key]}
-                            />
-                          ) : (
-                            col_configs_with_defaults[col_key].formatter(
-                              total_row[col_key]
-                            )
-                          )
+            </thead>
+            <tbody>
+              {_.map(sorted_filtered_data, (row, i) => (
+                <tr key={i}>
+                  {_.map(visible_ordered_col_keys, (col_key) => (
+                    <td style={{ fontSize: "14px" }} key={col_key}>
+                      {col_configs_with_defaults[col_key].formatter ? (
+                        _.isString(
+                          col_configs_with_defaults[col_key].formatter
+                        ) ? (
+                          <Format
+                            type={col_configs_with_defaults[col_key].formatter}
+                            content={row[col_key]}
+                          />
                         ) : (
-                          total_row[col_key]
+                          col_configs_with_defaults[col_key].formatter(
+                            row[col_key]
+                          )
                         )
                       ) : (
-                        ""
+                        row[col_key]
                       )}
                     </td>
-                  ))
-                  .value()}
-              </tr>
-            )}
-          </tbody>
-        </table>
+                  ))}
+                </tr>
+              ))}
+              {is_total_exist && (
+                <tr key="total_row">
+                  <td>{text_maker("total")}</td>
+                  {_.chain(visible_ordered_col_keys)
+                    .tail()
+                    .map((col_key) => (
+                      <td key={col_key}>
+                        {total_row[col_key] ? (
+                          col_configs_with_defaults[col_key].formatter ? (
+                            _.isString(
+                              col_configs_with_defaults[col_key].formatter
+                            ) ? (
+                              <Format
+                                type={
+                                  col_configs_with_defaults[col_key].formatter
+                                }
+                                content={total_row[col_key]}
+                              />
+                            ) : (
+                              col_configs_with_defaults[col_key].formatter(
+                                total_row[col_key]
+                              )
+                            )
+                          ) : (
+                            total_row[col_key]
+                          )
+                        ) : (
+                          ""
+                        )}
```

Comment:
Not a necessary change, but this ternary's gotten a bit big. Would suggest refactoring the as an immediately invoked function:
```
<td key={col_key}>
  {(()=>{
    const has_total_row = total_row[col_key];
    if (has_total_row){
      // various nested cases
    } else {
      return "";
    }
  })()}
</td>
```